systems/plants/RigidBodyManipulator.cpp

```diff
@@ -363,12 +363,12 @@ void RigidBodyManipulator::compile(void)
 {
   // reorder body list to make sure that parents before children in the list
 	size_t i=0;
-  while (i<bodies.size()) {
+  while (i<bodies.size()-1) {
```

Comment:
In #1054 we changed some `size_t` to `int`, should we also do it herein line 365?